### PR TITLE
app-containers/lxd: add 6.1

### PR DESCRIPTION
**Pull request by listout**
---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

**Comment by juippis** [comment]
I'll take a look at some point, but meanwhile could you bump dqlite to 1.16.5? Bundle it in this PR or make a new one, doesn't matter.

File: app-containers/lxd/lxd-6.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+	default
+
+	sed -i \
+		-e "s:\./configure:./configure --prefix=/usr --libdir=${EPREFIX}/usr/lib/lxd:g" \
+		-e "s:make:make ${MAKEOPTS}:g" \
+		Makefile || die
+
+	# Fix hardcoded ovmf file path, see bug 763180
+	sed -i \
+		-e "s:/usr/share/OVMF:/usr/share/edk2-ovmf:g" \
```
Comment (juippis) [inline app-containers/lxd/lxd-6.1.ebuild:100]:
```
 * SED: some expressions in the following did not cause any changes
 *     sed -i -e "s:/usr/share/OVMF:/usr/share/edk2-ovmf:g" -e "s:OVMF_VARS.ms.fd:OVMF_VARS.fd:g" doc/environment.md lxd/apparmor/instance.go lxd/apparmor/instance_qemu.go lxd/instance/drivers/driver_qemu.go || die "Failed to fix hardcoded ovmf paths.";
 * no-op: -e s:/usr/share/OVMF:/usr/share/edk2-ovmf:g
```
Could you check whether upstream updated these files with the path that we've used?

Comment (listout) [reply app-containers/lxd/lxd-6.1.ebuild:100]: It looks like the sed command is modifying lxd/instance/drivers/driver_qemu.go and doc/environment.md, from the command commit https://github.com/canonical/lxd/commit/a10fe694bc0d4df579fda0ce00cc8cabf0a037d4. That files looks like it's the same and in 6.1 tag

Comment (listout) [reply app-containers/lxd/lxd-6.1.ebuild:100]: Please correct me if I'm wrong.

Comment (juippis) [reply app-containers/lxd/lxd-6.1.ebuild:100]: Well no big reason to split the sed call into two, this is still fine.

**Review by juippis — APPROVED** [review]

File: app-containers/lxd/lxd-6.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+inherit bash-completion-r1 go-module linux-info optfeature systemd verify-sig
+
+DESCRIPTION="Modern, secure and powerful system container and virtual machine manager"
+HOMEPAGE="https://ubuntu.com/lxd https://github.com/canonical/lxd"
+SRC_URI="https://github.com/canonical/lxd/releases/download/${P}/${P}.tar.gz
+	verify-sig? ( https://github.com/canonical/lxd/releases/download/${P}/${P}.tar.gz.asc
+)"
+
+LICENSE="Apache-2.0 AGPL-3+ BSD LGPL-3 MIT"
+SLOT="0/lts"
```
Comment (juippis) [inline app-containers/lxd/lxd-6.1.ebuild:15]: Oh, one thing I forgot to check/ask: Is this true?

Comment (juippis) [reply app-containers/lxd/lxd-6.1.ebuild:15]: 8281cc21f66353112ac97c8bb8ccb4096db9cbe0

**Pull request closed without merging**